security/pkg/util/jwtutil.go

```diff
@@ -107,7 +107,7 @@ func ExtractJwtAud(jwt string) ([]string, bool) {
 	}
 	payload := jwtSplit[1]
 
-	payloadBytes, err := base64.RawStdEncoding.DecodeString(payload)
+	payloadBytes, err := base64.URLEncoding.DecodeString(payload)
```

Comment:
Thanks for your PR! Can you add a comment with the link to the JWT spec on the encoding method?

Reply:
Link to the JWT spec: https://www.rfc-editor.org/rfc/rfc7519.  All searches within this document for base64 will reveal that only base64url encoding is used.

Reply:
I think it worth a test

Reply:
@kellinwood Can you add a test?

Reply:
I can add a test.  Also, there are currently 6 failing tests so I think coverage is already pretty good.  I am in the process of figuring out how to run the verification/tests stuff locally.

Reply:
Code updated, test added.  To be crystal clear, The JWT spec (RFC7519) defers to the JWS spec (RFC7515) on the implementation of base64url encoding.  The JWS spec calls for `=` padding characters to be removed during encoding.  My latest push accounts for that in the istio jwt payload decoding.